### DISCUSSION TITLE
Demonstrate code examples for each setting?

### DISCUSSION THREAD

**Discussion by trymeouteh** [lkrms/pretty-php#124] [Ideas]
Some of the settings have code examples such as `Align Data`. Please have coding examples for every setting. I do not know what to set some of the settings to since I do not understand how they will change the formatting.

**Comment by lkrms** [comment]
I assume you're referring to the inline documentation offered when configuring the Visual Studio extension?

I try to add examples whenever I'm updating the extension, will endeavour to add some more 😊

**Reply by trymeouteh** [reply]
Thank you.
